File: _scratch/ScottStuff/en-us/channel-inspector/files/Facebook/Document/example1.cs
public async Task ExecuteCommand(ActivityContext activityContext, Activity activity)
{
	IMessageActivity iMessage = (IMessageActivity)activity;
	var reply = activity.CreateReply($"{activityContext.GetCounters()} {iMessage.Text}");
	reply.Attachments = reply.Attachments ?? new List<Attachment>();
	reply.Attachments.Add(new Attachment()
	{
		ContentType = "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
		ContentUrl = "https://kodu.blob.core.windows.net/kodu/KODU%20Help.docx",
		Name = "Kodu Help.docx"
	});

	await SendResponse(activityContext, reply).DropContext();
}

File: _scratch/ScottStuff/en-us/channel-inspector/files/Skype/Document/example1.js
function BotSendDocument(session, builder){
	var msg = new builder.Message(session).attachments([{
		name: "Kodu Help.docx",
		contentType: "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
		contentUrl: "https://kodu.blob.core.windows.net/kodu/KODU%20Help.docx"
	}]);
	session.send(msg);
}
